Directory.Build.props

```diff
@@ -7,7 +7,7 @@
 		<MicrosoftExtensionsHostingVersion>7.0.1</MicrosoftExtensionsHostingVersion>
 		<MicrosoftExtensionsConfigurationVersion>7.0.0</MicrosoftExtensionsConfigurationVersion>
 		<SystemCommandLineHostingVersion>0.3.0-alpha.20574.7</SystemCommandLineHostingVersion>
-		<SystemIOAbstractionsVersion>17.2.3</SystemIOAbstractionsVersion>
+		<SystemIOAbstractionsVersion>19.2.69</SystemIOAbstractionsVersion>
```

Comment:
I think it would be nice to change the variable name as well to fully match the package name

Reply:
PR is updated with the changes